assets/scripts/coordination/coordination_hist.py

```diff
@@ -8,7 +8,7 @@
 
 import pymatviz as pmv
 from pymatviz.coordination import CnSplitMode
-from pymatviz.utils import TEST_FILES
+from pymatviz.utils.testing import TEST_FILES
```

Comment:
maybe reexport `TEST_FILES` from `utils/__init__.py`

Reply:
I kind of want to hide test utils like this from the "public" interface (`pmv.utils`), what do you think?

Reply:
sounds good. i moved `pymatviz/utils/typing.py` to `pymatviz/typing.py` since i think types are important enough to not be relegated into `utils`. we also now match structure with `numpy.typing`

Reply:
That's a good idea, but would be breaking?

Reply:
good point! still, i think we should go ahead. i imagine very few people import `pymatviz` types in external code bases anyway.